types/joi-password-complexity/index.d.ts

```diff
@@ -1,24 +1,51 @@
-// Type definitions for joi-password-complexity 3.1
+// Type definitions for joi-password-complexity 4.2
 // Project: https://github.com/kamronbatman/joi-password-complexity
-// Definitions by: Nico Flaig <https://github.com/nflaig>
+// Definitions by: Kamron Batman <https://github.com/kamronbatman>
```

Comment:
@AlanMorel you are not supposed to remove other contributors...

Since you are contribution those changes you can add yourself or since its just copy/pasted from the [library index.d.ts file](https://github.com/kamronbatman/joi-password-complexity/blob/master/index.d.ts) maybe adding the author of that makes sense.

Reply:
Hey @nflaig, sorry about that, wasn't exactly sure if I needed to keep the old contributor if I was no longer using their work. I'm completely new to TypeScript and DefinitelyTyped, so my question is, since the library now includes their own `index.d.ts`, is it enough for the library to just bump its version one more time, thus allowing everybody who updates to get that new file, and then `@types/joi-password-complexity` would be no longer needed at all? If so, that sounds like the better route, and then the types can just be removed from DefinitelyTyped entirely instead.

cc: @BjMrq @kamronbatman

Reply:
I am ok with the update going directly to joi-password-complexity and deleting it from here.
Thanks @nflaig for the contribution.

Reply:
Same !

Reply:
Having the types directly in the package seems like the best solution, however I think there needs to be another definitly typed release which adds a deprecation note, see [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/26449/files) as an example

Reply:
Thanks everybody, I appreciate you all guiding me through this completely new process! I will close this PR now per Kamron's comment.